tests/components/deako/conftest.py

```diff
@@ -0,0 +1,31 @@
+"""deako session fixtures."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from homeassistant.components.deako.const import DOMAIN
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_config_entry() -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    return MockConfigEntry(
+        domain=DOMAIN,
+    )
+
+
+@pytest.fixture(name="pydeako_deako_mock", autouse=True)
+def pydeako_deako_mock():
+    """Mock pydeako deako client."""
+    with patch("homeassistant.components.deako.Deako", autospec=True) as mock:
+        yield mock
+
+
+@pytest.fixture(name="pydeako_discoverer_mock", autouse=True)
+def pydeako_discoverer_mock(mock_async_zeroconf: MagicMock):
```

Comment:
This is needed since it's a dep to the integration and the deako discoverer takes in a zeroconf instance and then subscribes to mDNS additions/updates/removals. Without this, I get this for almost all my tests:

`Unable to set up dependencies of 'deako'. Setup failed for dependencies: ['zeroconf']`

If there's a better way I should do this, I'm open to suggestions.